packages/client/src/views/UpcomingClosingDates.vue

```diff
@@ -213,9 +213,9 @@ export default {
         }
       }
       //                      format date in MM/DD/YY
-      const year = value.slice(2, 4);
```

Comment:
@misspran it looks like the error happens when the `value` is `undefined`.

My recommendation would be to update the top of the `formatDate` function in line 188. This way we can avoid all the unnecessary computation that this function does.
```js
formatDate(value) {
    if (!value) {
         // returns empty string without error if value is undefined
         return '';
    }
    ....rest of code stays the same
}
```